docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -314,7 +314,7 @@ Note that ownership cannot be used with polymorphic relations. E.g. has_one to n
 #### Unversioned dataobject ownership (4.1 and above)
 
 Ownership can be used with non-versioned dataobjects, as the necessary functionality is included by default
-by the versioned object through the `[api:SilverStripe\Versioned\RecursivePublishable]` extension which is
+by the versioned object through the (`RecursivePublishable`)[api:SilverStripe\Versioned\RecursivePublishable] extension which is
```

Comment:
isn't this the wrong way round? shouldn't it be 

```diff
-(`RecursivePublishable`)[api:SilverStripe\Versioned\RecursivePublishable]
+[`RecursivePublishable`](api:SilverStripe\Versioned\RecursivePublishable)
```

Reply:
Yep, that'll teach me for speed coding